### PR TITLE
Added the capability to use icon inside a badge

### PR DESCRIPTION
**Pull request by GBT3101**

### CODE REVIEW

File: src/assets/icons/Camera.svg
Comment (jonathanadler) [inline src/assets/icons/Camera.svg:10]: Please remove all the `id`'s

File: src/components/Badge/Badge.driver.ts
Comment (jonathanadler) [inline src/components/Badge/Badge.driver.ts:29]: Using the class name `.icon` won't necessary work, since it depends on the hashing that's used on the compiled bundle.
So instead of classes please use a `data-` hook (like here: https://github.com/wix/wix-ui-tpa/pull/376/files#diff-da702d7b76d4fd862bd1a8e985077d14R12)

File: src/components/Badge/Badge.st.css
Comment (jonathanadler) [inline src/components/Badge/Badge.st.css:30]: It seems you're not using the `withIcon` state.
So you can just remove it

File: src/components/Badge/Badge.tsx
Comment (jonathanadler) [inline src/components/Badge/Badge.tsx:33]: We prefer to not use `style` attributes in our components.
So please use a class.
btw, is this `div` really needed, or is the root `div` enough?

**Review by jonathanadler — COMMENTED** [review]
Great work!

Just left some comments.
Also, please add a visual test (with an icon) here: https://github.com/wix/wix-ui-tpa/blob/master/src/components/Badge/Badge.visual.tsx

Thanks!

File: src/components/Badge/Badge.driver.ts
Comment (jonathanadler) [inline src/components/Badge/Badge.driver.ts:24]: so I think we should change this to `hasIcon` cause we're not really returning the icon, but a boolean

**Review by jonathanadler — COMMENTED** [review]
Nice!
Just left a really really small comment.
I'm also waiting for the designers to approve
and after that we can merge 💪 
Great work
thanks!

File: src/components/Badge/Badge.driver.ts
Comment (GBT3101) [reply src/components/Badge/Badge.driver.ts]: You are right, changed it, I had to change the inner "hasIcon" function because it would shadow it, changed the inner one to "isIconExists"

**Review by jonathanadler — APPROVED** [review]
Great!
I just need to get a confirmation from the designers and then we can merge 💪 
Thanks for the contribution!

**Pull request merged**
